src/etc/man/cargo-vendor.1

```diff
@@ -54,13 +54,21 @@ vendored and synced to the output.
 \fB\-\-no\-delete\fP
 .RS 4
 Don\(cqt delete the "vendor" directory when vendoring, but rather keep all
-existing contents of the vendor directory
+existing contents of the vendor directory.
+.RE
+.sp
+\fB\-\-versioned\-dirs\fP
+.RS 4
+Normally versions are only added to disambiguate multiple versions of the same package.
+This option causes all directories in the "vendor" directory to be versioned,
+which makes it easier to track the history of vendored packages over time, and can help
+with the performance of re-vendoring when only a subset of the packages have changed.
```

Comment:
This needs to go in the `.adoc` file, otherwise it will be lost when the documentation is regenerated.  There is some documentation at https://github.com/rust-lang/cargo/tree/master/src/doc on how the docs are generated.